src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -59,9 +62,13 @@ public function whenSavingUser(Saving $event)
 
             $this->assertCan($actor, 'suspend', $user);
 
-            $user->suspend_until = $attributes['suspendUntil']
-                ? new DateTime($attributes['suspendUntil'])
-                : null;
+            if (isset($attributes['suspendUntil'])) {
```

Comment:
Shouldn't this make sure that there is a change here? If I PATCH the `suspendUntil` to `null` - and it was `null` previously, then surely the event should not be raised?

Reply:
Yeah, it shouldn't be.
Would `if ($attributes->suspendUntil == $user->suspendUntil) return;` be sufficient? I am unable to test because of the master changes, I'll have to create another installation with master soon.

Reply:
Yeah, sound's good. Let's give it a `===` comparison, though.

Feel free to ping me on Discord - would be interested to hear what problems you're having (error message?) with getting the current master to run.